hypothesisfuzzer/fuzzing/repo_fuzzer.py

```diff
@@ -60,13 +59,18 @@ def get_errors(self):
             return jsonify(json.load(file_data))
 
     def _clone_git(self, git_url):
-        # Delete old code folder
 
+        # Pull repository if already exists
         if os.path.exists(self.name):
-            shutil.rmtree(self.name, ignore_errors=True)
-
-        os.makedirs(self.name)
-        GitRepo.clone_from(git_url, self.name)
+            # self.log.debug('Updating existing repository %s.', self.name)
+            GitRepo(self.name).git.pull()
```

Comment:
You need to git reset --hard origin/<branch> first, in order to prevent pull errors if the local files have been manually changed.

Also might need to wrap the git pull in a try-catch block or handle the error in some other way because git pull only works via ssh and this needs the user to have configured ssh keys.

Reply:
Added it anyway. Might be an issue if pulling from branch that isn't master.